common/lib/xmodule/xmodule/js/spec/video/video_events_plugin_spec.js

```diff
@@ -0,0 +1,166 @@
+            $('source').remove();
+            window.onTouchBasedDevice = oldOTBD;
+            state.storage.clear();
+            if (state.videoPlayer) {
+                state.videoPlayer.destroy();
+            }
+        });
+
+        it('can emit "load_video" event', function () {
+            state.el.trigger('ready');
+            expect(Logger.log).toHaveBeenCalledWith('load_video', {
+                id: 'id',
+                code: 'html5'
+            });
+        });
+
+        it('can emit "play_video" event', function () {
+            state.el.trigger('play');
+            expect(Logger.log).toHaveBeenCalledWith('play_video', {
+                id: 'id',
+                code: 'html5',
+                currentTime: 10
+            });
+        });
+
+        it('can emit "pause_video" event', function () {
+            state.el.trigger('pause');
+            expect(Logger.log).toHaveBeenCalledWith('pause_video', {
+                id: 'id',
+                code: 'html5',
+                currentTime: 10
+            });
+        });
+
+        it('can emit "speed_change_video" event', function () {
+            state.el.trigger('speedchange', ['2.0', '1.0']);
+            expect(Logger.log).toHaveBeenCalledWith('speed_change_video', {
+                id: 'id',
+                code: 'html5',
+                current_time: 10,
+                old_speed: '1.0',
+                new_speed: '2.0'
+            });
+        });
+
+        it('can emit "seek_video" event', function () {
+            state.el.trigger('seek', [1, 0, 'any']);
+            expect(Logger.log).toHaveBeenCalledWith('seek_video', {
+                id: 'id',
+                code: 'html5',
+                old_time: 0,
+                new_time: 1,
+                type: 'any'
+            });
+        });
+
+        it('can emit "stop_video" event', function () {
+            state.el.trigger('ended');
+            expect(Logger.log).toHaveBeenCalledWith('stop_video', {
+                id: 'id',
+                code: 'html5',
+                currentTime: 10
+            });
+
+            Logger.log.reset();
+            state.el.trigger('stop');
+            expect(Logger.log).toHaveBeenCalledWith('stop_video', {
+                id: 'id',
+                code: 'html5',
+                currentTime: 10
+            });
+        });
+
+        it('can emit "skip_video" event', function () {
+            state.el.trigger('skip', [false]);
+            expect(Logger.log).toHaveBeenCalledWith('skip_video', {
+                id: 'id',
+                code: 'html5',
+                currentTime: 10
+            });
+        });
+
+        it('can emit "do_not_show_again_video" event', function () {
+            state.el.trigger('skip', [true]);
+            expect(Logger.log).toHaveBeenCalledWith('do_not_show_again_video', {
+                id: 'id',
+                code: 'html5',
+                currentTime: 10
+            });
+        });
```

Comment:
> It would be really helpful to have output for this new event also. Thanks Alex.

Please see https://github.com/edx/edx-platform/pull/7825#issuecomment-101942782